Source/Constraint.swift

```diff
@@ -417,27 +407,15 @@ public class Constraint {
     
     internal func uninstallFromView() {
         if let view = self.installedOnView {
-            // remove all installed layout constraints
-            var layoutConstraintsToRemove = Array<LayoutConstraint>()
-            if let allObjects = self.installedLayoutConstraints?.allObjects {
-                if let installedLayoutConstraints = allObjects as? Array<LayoutConstraint> {
-                    layoutConstraintsToRemove += installedLayoutConstraints
-                }
-            }
-            
-            if layoutConstraintsToRemove.count > 0 {
+            if let layoutConstraintsToRemove = self.installedLayoutConstraints?.allObjects {
                 view.removeConstraints(layoutConstraintsToRemove)
             }
-            
-            // clean up the snp_installedLayoutConstraints
-            var layoutConstraints = view.snp_installedLayoutConstraints
-            var layoutConstraintsToKeep = Array<LayoutConstraint>()
-            for layoutConstraint in layoutConstraints {
-                if !contains(layoutConstraintsToRemove, layoutConstraint) {
-                    layoutConstraintsToKeep.append(layoutConstraint)
+
+            if let installedConstraints = self.fromItem.view?.snp_installedLayoutConstraints {
```

Comment:
This!. On the old code, the constraints were trying to be removed based on `view.snp_installedLayoutConstraints` which wasn't doing anything. This fixes a constraint leak that exponentially slows down a view with many `remakeconstraint`s